.github/workflows/update-readme.yml

```diff
@@ -17,13 +17,15 @@ jobs:
         run: |
           sudo apt-get update
           sudo add-apt-repository universe
-          wget https://github.com/neovim/neovim/releases/download/nightly/nvim.appimage
+          wget https://github.com/neovim/neovim/releases/download/v0.5.0/nvim.appimage
```

Comment:
can you create a separate PR for that. Please also use v0.5.1

Reply:
Don't worry. We don't have to overcomplicate things. Just update the version to v0.5.1